addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -122,10 +122,17 @@ var KanbanRenderer = BasicRenderer.extend({
      * Called each time the renderer is attached into the DOM.
      */
     on_attach_callback: function () {
+        _.invoke(this.widgets, 'on_attach_callback');
```

Comment:
doesn't it crash on grouped kanban views?

Reply:
No, `this.widgets` still an array in grouped kanban views (array with single element).

Reply:
Sure, but isn't it an array of KanbanColumn instances?

Reply:
Correct, 'on_attach_callback' is not defined on them, and `_.invoke` silently fails on them, as you can see with its implementation:

https://github.com/odoo/odoo/blob/master/addons/web/static/lib/underscore/underscore.js#L268

```
_.invoke = function(obj, method) { /* this.widgets, 'on_attach_callback' */
    var args = slice.call(arguments, 2);
    var isFunc = _.isFunction(method); /* false */
    return _.map(obj, function(value) {
      var func = isFunc ? method : value[method]; /* func = undefined */
      return func == null ? func : func.apply(value, args); /* return undefined */
    }); /* return [undefined] */
  };
```

Reply:
ok then :+1: